qdrant_test/image_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
-const TestImage string = "qdrant/qdrant:v1.12.0"
+const TestImage string = "qdrant/qdrant:dev"
```

Comment:
We need to update this on release:

```suggestion
const TestImage string = "qdrant/qdrant:v1.13.0"
```